src/components/Errors/ErrorBoundary.stories.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+
+import {PermissionDenied, ServiceUnavailable, UnprocessableEntity} from 'errors';
+
+import ErrorBoundary from './ErrorBoundary';
+
+const render = ({useCard, errorType, errorCode}) => {
+  const error = new errorType('some error', 500, 'some error', errorCode);
+
+  return (
+    <ErrorBoundary useCard={useCard}>
+      {React.createElement(() => {
+        throw error;
+      })}
+    </ErrorBoundary>
+  );
+};
+
+export default {
+  title: 'Private API / ErrorBoundary',
+  component: ErrorBoundary,
+  argTypes: {
+    useCard: {control: {type: 'boolean'}},
+    errorType: {
+      table: {
+        options: [PermissionDenied, ServiceUnavailable, UnprocessableEntity],
+        control: {type: 'radio'},
+      },
+    },
+  },
+};
+
+export const GenericError = {
+  render,
+  args: {
+    useCard: true,
+    errorType: Error,
+    errorCode: 'generic',
+  },
+};
+
+// FIXME cant get this to work with routerprovider
```

Comment:
This requires a Router because `<Link>` is used in the `PermissionDeniedError` component, but whenever I tried using the `RouterProvider` in Storybook it kept crashing the page...

Reply:
CTRL + F on `import {withRouter} from 'storybook-addon-remix-react-router';` in the codebase :) there's a Storybook decorator that sets up a router provider so you can render components that use hooks/links etc.

Reply:
It does work if I open the story one, but if I then switch to another story and go back to this story, it keeps crashing :/

Reply:
I'll take a look :)